### PR TITLE
add indonesian translation

### PR DESCRIPTION
**Pull request by adityapurwa**
Added Indonesian translation to make it easier to understand for Indonesian developers, I will ask for another native speaker to help me validating this request.

### CODE REVIEW

File: source/id-ID/1.0.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    dan kenapa terjadi perubahan.
+
+.good-practices
+  %h3#how
+    %a.anchor{ href: "#how", aria_hidden: "true" }
+    Bagaimana cara membuat changelog yang baik?
+
+  %h4#principles
+    %a.anchor{ href: "#principles", aria_hidden: "true" }
+    Prinsip Dasar
```
Comment (cyberid41) [inline source/id-ID/1.0.0/index.html.haml]: What about the `Prinsip - Prinsip Dasar`

Comment (adityapurwa) [reply source/id-ID/1.0.0/index.html.haml]: I thinks that's better as it is plural, will update.

**Comment by cyberid41** [comment]
Greate job @adityapurwa , keep the spirit ! 👍

**Comment by olivierlacan** [comment]
@adityapurwa @cyberid41 Let me know when it's good to merge. 😃

**Comment by adityapurwa** [comment]
@olivierlacan from my side is good, all parts has been translated.

**Comment by olivierlacan** [comment]
Thanks a lot to both of you! 💥

**Comment by olivierlacan** [comment]
There you go: http://keepachangelog.com/id-ID/1.0.0/

**Comment by adityapurwa** [comment]
If there are any further update on the translations, will it be automatically be published when I pull request to master, or do I have to submit a pull request for the gh-pages branch?

**Comment by olivierlacan** [comment]
@adityapurwa No, I publish them myself. You don't have to worry about that. I usually batch releases to GitHub Pages.

**Pull request merged**